### DISCUSSION TITLE
Is there any way to change voice in gtts?

### DISCUSSION THREAD

**Discussion by ns696** [pndurette/gTTS#299] [Q&A] [answered]
Is there any way to change voice in gtts?

**Comment by pndurette** [comment, marked as answer]
There's a way to influence the voice, but it's a limited and still not guaranteed gender. See [Localized 'Accents'](https://gtts.readthedocs.io/en/latest/module.html#localized-accents).

**Reply by erik-erlandsson** [reply]
Do you have any plans to implement the "Voice type" and "Voice name" API calls?

**Reply by pndurette** [reply]
> Do you have any plans to implement the "Voice type" and "Voice name" API calls?

It's a whole different API as far as I know—this one is way more basic and non configurable unfortunately.

**Comment by ns696** [comment]
Thanks :-)

**Comment by erik-erlandsson** [comment]
Ok, I see.
 Thanks for your fast response!

Merry Christmas!

/erik

Den fre 17 dec. 2021 17:24Pierre Nicolas Durette ***@***.***>
skrev:
